domains/raikan.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "raikan-san",
+           "email": "",
+           "discord": "459219626484826113",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.ZcGtou9b6v4gRzrKJfeAQi2Igj-Dkrj0wK_13yM-ojuXdkJCDYTg74sz115KxObcHMG8vwsCW8S-Wd6ES67fU6qKpvzELQZ8AKBDaVcdyGllrCPkVm-iuSvpKtnkzZ4Y_MQEyMHKUK5GKtOWDeLVNpDYQjubihOcN0AScEljUKNAA_fkyENNr2uQsR2xKlJBV6OyFjDjN37hNYQiIgi7BJ6eJ7UWs67jCu_x5GZzvKNpJM-xJalJuDBtCxJvqZnv6RMRNg8_a2D8NZP_E8K_n77dkdRGdTNl5DpaM25q5xpk2oycci-9ViYl8tO1FF2iuScUJ6av6q6NSM4vHkOOlA.KCsD01AM71qG5XC2BlzCDw.-IMSZFWEnZ3GpahXrPSkjFcohMleT2YgUh4f3yXuXG-d6ZD-nzszGJrHu8P9dpoN_U1snAzGKJ5ANaQ4Ci4Zrnc5mavtQoaXDd5aLStmlSaStkL7olGkr-c2WP7UgcTZ.R1m60Waa-jCtLvhAAdFIxg"
+        },
+    
+        "record": {
+            "CNAME": "raikan.is-a.dev"
```

Comment:
The CNAME value is the same as the domain which you are requesting which is invalid.